abjad/tools/schemetools/Scheme.py

```diff
@@ -326,18 +331,20 @@ def format_scheme_value(value, force_quotes=False, verbatim=False):
             else:
                 return value
         elif isinstance(value, bool):
-            if value:
+            if value is True:
                 return '#t'
-            return '#f'
+            else:
```

Comment:
I was taking the 'explicit is better than implicit' approach, but if you prefer I'll revert those explicit `else` cases and sim and keep that in mind in the future.